### ISSUE TITLE
Bug get Fields 

### ISSUE THREAD

**Issue by KORCHID** [PrestaShop/PrestaShop#23628]
#### Describe the bug

some attributes return the default declared value 
#### Expected behavior

For example `show_condition` return 0 always
#### Steps to Reproduce

My temporary solution:

1.  add a new function to `ProductCore`
   ```
 public function getShowCondition($id_product) {
        $sql = 'SELECT `show_condition`
        FROM `' . _DB_PREFIX_ . 'product`
        WHERE `id_product` = ' . (int) $id_product;
        return (int) Db::getInstance(_PS_USE_SQL_SLAVE_)->getValue($sql);
    }
```
   

2. call the function in `__construct`
`            $this->show_condition = $this->getShowCondition($id_product);
`
**Screenshots**

![Capture du 2021-03-12 16-49-05](https://user-images.githubusercontent.com/33570600/110985964-612dd680-836d-11eb-9da2-69d62b56e169.png)
![Capture du 2021-03-12 12-18-20](https://user-images.githubusercontent.com/33570600/110985966-61c66d00-836d-11eb-995c-734c250bc4f1.png)

**after temporary solution**
![Capture du 2021-03-12 20-10-44](https://user-images.githubusercontent.com/33570600/110987218-101ee200-836f-11eb-8a0e-be05bb8ac8ff.png)

**Comment by hibatallahAouadni** [comment]
Hello @KORCHID 

Could you please provide steps to reproduce the issue and not steps of your solution :wink: 
And please provide your shop version and your PHP version.

For your PR, you need to fill the provided table with your PR informations., for exemple:
For **Category?** => **CO** (it means the change is in the Core files)
For **Fixed ticket?** => #23628 (this issue id)
Please check and feedback.

Thanks!

**Comment by hibatallahAouadni** [comment]
Hello @KORCHID ,

Since we had no news from you for more than 20 days, I'll close this ticket to avoid cluttering up the backlog.
Please note that you can always create a new one if further information pops up.

Thanks!
